core/src/main/java/com/datastax/oss/driver/internal/core/cql/CqlRequestHandlerBase.java

```diff
@@ -575,18 +598,17 @@ private void processErrorResponse(Error errorMessage) {
                             || prepareError instanceof FunctionFailureException
                             || prepareError instanceof ProtocolError) {
                           LOG.trace("[{}] Unrecoverable error on reprepare, rethrowing", logPrefix);
-                          trackNodeError(node, prepareError);
+                          trackNodeError(node, prepareError, -1);
                           setFinalError(prepareError, node, execution);
                           return null;
                         }
                       }
                     } else if (exception instanceof RequestThrottlingException) {
-                      trackNodeError(node, exception);
```

Comment:
Not directly related to this ticket, but we don't need to track the error here since throttling happens on the client.